### PR TITLE
[WIP] skip the tests that are not supposed to work with docker

### PR DESCRIPTION
**Pull request by jsuchome**
Not tested yet, just preliminary idea.
I'm not even sure if this is the right set of tests to exclude

### CODE REVIEW

**Comment by dirkmueller** [comment]
I suggest to disable the volumes extensions in the compute api extensions of tempest.conf to skip those tests. -1

**Comment by jsuchome** [comment]
@dirkmueller is it
``[compute-feature-disabled]
api_extensions="..."``
?
But I have no idea what are the possible values. Just "volumes" ?

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2398,6 +2398,10 @@ function oncontroller_testsetup()
         crudini --set /etc/tempest/tempest.conf orchestration image_ref $imageid
         pushd /var/lib/openstack-tempest-test
         echo 1 > /proc/sys/kernel/sysrq
+        if [ -n "$want_docker" ] ; then
+            testr list-tests | grep -v "tempest.api.compute.volumes" > docker_tests_excluding_volumes
+            export tempestoptions+="-- --load-list docker_tests_excluding_volumes"
+        fi
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: Could this go into barclamp-tempest instead?

Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: Yes, once I realize what should be in tempest.conf

**Comment by jsuchome** [comment]
apparently there's some better solution already by @aplanas , so closing this one

**Pull request closed without merging**